### PR TITLE
docs(effects): add the effects lifecycle page

### LINKED ISSUE #1456: Freeze chrome
## Minimal reproduction of the bug/regression with instructions:
Next code freeze chrome:
@Effect() public breakMe$ = this.actions$.pipe();
No errors, no warnings;

## Expected behavior:
No freeze;

## Versions of NgRx, Angular, Node, affected browser(s) and operating system(s):
@ngrx/effects": "6.1.2"
@angular/core": "7.1.0",
Google Chrome: 70.0.3538.110
## I would be willing to submit a PR to fix this issue

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[ +] No

### PR DESCRIPTION
**Pull request by Teamop**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #1456

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/effects/api.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
+# API
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/effects/api.md]: Change to Lifecycle

File: projects/ngrx.io/content/guide/effects/api.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+## EffectsModule
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/effects/api.md]: Remove this section as it should be in the API docs

File: projects/ngrx.io/content/guide/effects/api.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+
+```ts
+@Effect()
+init$ = this.actions$.pipe(
+  ofType(ROOT_EFFECTS_INIT),
+  map(action => ...)
+);
+```
+
+### forFeature
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/effects/api.md]: Remove this section

File: projects/ngrx.io/content/guide/effects/api.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+
+Registers @ngrx/effects services to run with your feature modules.
+
+**Note**: Running an effects class multiple times, either by `forRoot()` or `forFeature()`, (for example via different lazy loaded modules) will not cause Effects to run multiple times. There is no functional difference between effects loaded by `forRoot()` and `forFeature()`; the important difference between the functions is that `forRoot()` sets up the providers required for effects.
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/effects/api.md]: Move this note to the bottom of the Overview page

File: projects/ngrx.io/content/guide/effects/api.md [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+
+Usage:
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/effects/api.md]: Remove usage example

File: projects/ngrx.io/content/guide/effects/api.md [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+import { Injectable } from '@angular/core';
+import { Actions } from '@ngrx/effects';
+
+@Injectable()
+export class SomeEffectsClass {
+  constructor(private actions$: Actions) {}
+}
+```
+
+### ofType
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/effects/api.md]: Remove this section

File: projects/ngrx.io/content/guide/effects/api.md [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+        resolvedEffects$.pipe(
+          takeUntil(this.actions$.pipe(ofType('LOGGED_OUT')))
+        )
+      )
+    );
+  }
+}
+```
+
+## Utilities
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/effects/api.md]: Remove this section

File: projects/ngrx.io/content/navigation.json [hunk 8]
```diff
@@ -131,6 +131,10 @@
         {
           "title": "Testing",
           "url": "guide/effects/testing"
+        },
+        {
+          "title": "API",
+          "url": "guide/effects/api"
```
Comment (brandonroberts) [inline projects/ngrx.io/content/navigation.json]: change to guide/effects/lifecycle

**Comment by Teamop** [comment]
done, thanks @brandonroberts for your review

File: projects/ngrx.io/content/guide/effects/lifecycle.md [hunk 9]
```diff
@@ -0,0 +1,183 @@
+# Lifecycle
+
+### forRoot
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/effects/lifecycle.md]: Remove this section as its already in the overview

File: projects/ngrx.io/content/guide/effects/lifecycle.md [hunk 10]
```diff
@@ -0,0 +1,183 @@
...
+```ts
+@Effect()
+init = this.actions.pipe(
+  ofType<UpdateEffects>(UPDATE_EFFECTS)
+  filter(action => action.effects.includes('SomeEffectsClass')),
+  map(action => ...)
+);
+```
+
+## Actions
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/effects/lifecycle.md]: Remove this section

File: projects/ngrx.io/content/guide/effects/index.md [hunk 11]
```diff
@@ -89,3 +89,5 @@ import { AdminEffects } from './effects/admin.effects';
 })
 export class AdminModule {}
 ```
+
+**Note**: Running an effects class multiple times, either by `forRoot()` or `forFeature()`, (for example via different lazy loaded modules) will not cause Effects to run multiple times. There is no functional difference between effects loaded by `forRoot()` and `forFeature()`; the important difference between the functions is that `forRoot()` sets up the providers required for effects.
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/effects/index.md]: One additional change is to wrap this in a div

```html
<div class="alert is-important">

Note content here with line break above and below.

</div>

**Comment by Teamop** [comment]
done, thanks again @brandonroberts

**Comment by brandonroberts** [comment]
Thanks @Teamop!

**Pull request merged**